CreateSurfaceWindowsImage.ps1

```diff
@@ -806,12 +854,23 @@ Function Get-LatestUpdates
         $Cumulative = $False,
         $CumulativeDotNet = $False,
         $Adobe = $False,
+        $OOB = $False,
+        $Windows,
         $Path,
         $Date,
         $OSBuild,
         $Architecture
     )
 
+    If ($Windows -eq "W10")
+    {
+        $Windows = "Windows 10"

```

Comment:
Instead of re-editing variable inputs (which gets really confusing), it would be better to assign this to a new variable

It looks like $global:WindowsVersion is used to set the W10 or W11 strings

I'm also confused as to why globals are used but then also passed as arguments (if they're globals, why do they need to be passed?) Although I guess that's an issue throughout the script...